src/controllers/media.ts

```diff
@@ -64,3 +74,40 @@ export const getByOsdbHash = asyncHandler(async(req: Request, res: Response) =>
   dbMeta = await MediaMetadata.create(newMetadata);
   return res.json(dbMeta);
 });
+
+export const getBySanitizedTitle = asyncHandler(async(req: Request, res: Response, next: NextFunction) => {
+  const { title, language = 'eng' } = req.body;
+
+  if (!title) {
+    return next(new Error('title is required'));
```

Comment:
If you change this line to `throw new Error('title is required');` does it still go through the `next` function or do we need to explicitly do this?

Reply:
`thrown` errors are passed to the Express error handler, so same same, but this is probably cleaner so updated